dotcom-rendering/cdk/bin/cdk.ts

```diff
@@ -11,6 +11,23 @@ const sharedProps = {
 	region: 'eu-west-1',
 };
 
+const cpuScalingSteps = {
+	scalingStepsOut: [
+		// No scaling up effect when p90 CPU is lower than 70%
+		{ lower: 0, upper: 70, change: 0 },
+		// When p90 CPU is higher than 70% we scale up by 50%
+		{ lower: 70, change: 50 },
+		// When p90 CPU is higher than 90% we scale up by 80%
+		{ lower: 90, change: 80 },
+	],
+	scalingStepsIn: [
+		// No scaling down effect when p90 CPU is higher than 70%
+		{ lower: 70, change: 0 },
+		// When p90 CPU is lower than 70% we scale down by 1
+		{ upper: 70, lower: 0, change: -1 },
+	],
```

Comment:
The current bounds are an estimate based on recent spikes but happy to take suggestions. 

We will of course monitor and adjust if required.